ansible_runner/display_callback/callback/awx_display.py

```diff
@@ -409,6 +409,7 @@ def set_task(self, task, local=False):
             task=(task.name or task.action),
             task_uuid=str(task._uuid),
             task_action=task.action,
+            resolved_action=getattr(task, 'resolved_action', ''),
```

Comment:
Any reason not to make the default value here `task_action` instead of empty string? `resolved_action` isn't guaranteed to be a module/action FQCN if the actual module run was a legacy library module anyway, so in the "old core" case where the value is missing, it seems like presenting a less-accurate value is preferable to no value at all...

Reply:
@nitzmahone I think both options should be ok. The processing side will likely do a regexp check for FQCN format on resolved_action, if it won't match we'll use task_action.

Reply:
The only option that could make more sense to me would be to exclude `resolved_action` if it's the same as `task`.

I pushed a commit that does the original suggestion here, just fallback to the `task.action`.